Binary files PDO_DBLIB-1.0.orig/.config.m4.swp and PDO_DBLIB-1.0/.config.m4.swp differ
diff -uriN PDO_DBLIB-1.0.orig/config.m4 PDO_DBLIB-1.0/config.m4
--- PDO_DBLIB-1.0.orig/config.m4	2005-11-01 07:12:52.000000000 -0600
+++ PDO_DBLIB-1.0/config.m4	2017-12-04 11:32:31.000000000 -0600
@@ -24,10 +24,6 @@
       fi
     done
 
-    if test -z "$PDO_FREETDS_INSTALLATION_DIR"; then
-      AC_MSG_ERROR(Cannot find FreeTDS in known installation directories)
-    fi
-
   elif test "$PHP_PDO_DBLIB" != "no"; then
 
     if test -f $PHP_PDO_DBLIB/include/tds.h; then
@@ -36,8 +32,6 @@
     elif test -f $PHP_PDO_DBLIB/include/freetds/tds.h; then
       PDO_FREETDS_INSTALLATION_DIR=$PHP_PDO_DBLIB
       PDO_FREETDS_INCLUDE_DIR=$PHP_PDO_DBLIB/include/freetds
-    else
-      AC_MSG_ERROR(Directory $PHP_PDO_DBLIB is not a FreeTDS installation directory)
     fi
   fi  
 
@@ -45,7 +39,7 @@
     PHP_LIBDIR=lib
   fi
 
-  if test ! -r "$PDO_FREETDS_INSTALLATION_DIR/$PHP_LIBDIR/libtds.a" && test ! -r "$PDO_FREETDS_INSTALLATION_DIR/$PHP_LIBDIR/libtds.so"; then
+  if test ! -r "$PDO_FREETDS_INSTALLATION_DIR/$PHP_LIBDIR/libtds.a" && test ! -r "$PDO_FREETDS_INSTALLATION_DIR/$PHP_LIBDIR/libtds.so" && test ! -r "$PDO_FREETDS_INSTALLATION_DIR/$PHP_LIBDIR/libtdsodbc.so"; then
      AC_MSG_ERROR(Could not find $PDO_FREETDS_INSTALLATION_DIR/$PHP_LIBDIR/libtds.[a|so])
   fi
 
